specs/rollup-node.md

```diff
@@ -146,6 +146,45 @@ Batch contents:
 
 [EIP-2718]: https://eips.ethereum.org/EIPS/eip-2718
 
+The guarantee must be held that all transactions containing batches
+being sent to L1 must be valid. This means that users must not be able
+to submit transactions to L2 that make an L1 transaction including it
+be invalid. The only way to do so would be to send a transaction to L2
+that is so large that it would use more than the L1 block's gas limit
+as part of [intrinsic gas][intrinsic-gas]. As of [London][london],
+a zero byte of calldata costs 4 gas while a non zero byte of calldata
+costs 16 gas and the [block gas limit is 30 million][block-size].
+An entire block's worth of gas could be used by a transaction with
+7.5 megabytes of zero byte calldata. Including a transaction
+that uses more than the entire block gas limit as part of it's
+intrinsic gas would cause a liveness failure of L2.
```

Comment:
Previously there was a concept of "force inclusion period" where if a deposit was not ingested in some time, it could be forcefully ingested by a user. This functionality was never really built out, but it would enable a fault proof in the case of a liveliness failure like mentioned here. What is the security model around a sequencer including a L2 tx that is too large to be included in a L1 block?